.gitignore

```diff
@@ -24,3 +24,8 @@ Cargo.lock
 # Added by cargo
 
 /target
+
+# Output file
+output.txt
```

Comment:
I'd suggest to move the output-related stuff from the software layer to products/outputs directory and exclude it completely in gitignore instead of individual files exclusion.

Reply:
Done